packages/vitepress-plugin-og-image-generator/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@nolebase/vitepress-plugin-og-image-generator",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "A vitepress plugin to generate og:image for your site",
+  "author": {
+    "name": "Nólëbase",
+    "email": "nolebase@ayaka.moe",
+    "url": "https://github.com/nolebase"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "README.md",
+    "dist",
+    "package.json"
+  ],
+  "scripts": {
+    "build": "unbuild"
```

Comment:
Since this is just a simple TypeScript module, is `stub` script possible to be here?